File: IntegrationFinale/User/src/entity/Chaise.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package entity;

/**
 *
 * @author asus
 */
public class Chaise {
    
    private int id_chaise;
    private int id_enfant;
    private String etat_plat;
    private int num_table;

    public Chaise(int id_chaise, int id_enfant, String etat_plat, int num_table) {
        this.id_chaise = id_chaise;
        this.id_enfant = id_enfant;
        this.etat_plat = etat_plat;
        this.num_table = num_table;
    }
    public Chaise(int id_chaise, String etat_plat, int num_table) {
        this.id_chaise = id_chaise;
        this.etat_plat = etat_plat;
        this.num_table = num_table;
    }


    public Chaise(String etat_plat, int num_table) {
        this.etat_plat = etat_plat;
        this.num_table = num_table;
    }
    
   
    public int getId_chaise() {
        return id_chaise;
    }

    public void setId_chaise(int id_chaise) {
        this.id_chaise = id_chaise;
    }

    public int getId_enfant() {
        return id_enfant;
    }

    public void setId_enfant(int id_enfant) {
        this.id_enfant = id_enfant;
    }

    public String getEtat_plat() {
        return etat_plat;
    }

    public void setEtat_plat(String etat_plat) {
        this.etat_plat = etat_plat;
    }

    public int getNum_table() {
        return num_table;
    }

    public void setNum_table(int num_table) {
        this.num_table = num_table;
    }

    @Override
    public String toString() {
        return "Chaise{" + "id_chaise=" + id_chaise + ", id_enfant=" + id_enfant + ", etat_plat=" + etat_plat + ", num_table=" + num_table + '}';
    }
    
    
}

File: IntegrationFinale/PiDevF2/PiDev/src/PidevBundle/Entity/Chaise.php
<?php

namespace PidevBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Chaise
 *
 * @ORM\Table(name="chaise", indexes={@ORM\Index(name="fk_enfant_chaise", columns={"id_enfant"}), @ORM\Index(name="fk_table_chaise", columns={"num_table"})})
 * @ORM\Entity(repositoryClass="PidevBundle\Repository\ChaiseRepository")
 */
class Chaise
{
    /**
     * @var integer
     *
     * @ORM\Column(name="id_chaise", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $idChaise;

    /**
     * @var string
     *
     * @ORM\Column(name="etat_plat", type="string", length=200, nullable=false)
     */
    private $etatPlat;

    /**
     * @var \Enfant
     *
     * @ORM\ManyToOne(targetEntity="Enfant")
     * @ORM\JoinColumns({
     *   @ORM\JoinColumn(name="id_enfant", referencedColumnName="id_enfant")
     * })
     */
    private $idEnfant;

    /**
     * @var \TableCantine
     *
     * @ORM\ManyToOne(targetEntity="TableCantine")
     * @ORM\JoinColumns({
     *   @ORM\JoinColumn(name="num_table", referencedColumnName="Id_table")
     * })
     */
    private $numTable;

    /**
     * @return int
     */
    public function getIdChaise()
    {
        return $this->idChaise;
    }

    /**
     * @param int $idChaise
     */
    public function setIdChaise($idChaise)
    {
        $this->idChaise = $idChaise;
    }

    /**
     * @return string
     */
    public function getEtatPlat()
    {
        return $this->etatPlat;
    }

    /**
     * @param string $etatPlat
     */
    public function setEtatPlat($etatPlat)
    {
        $this->etatPlat = $etatPlat;
    }

    /**
     * @return \Enfant
     */
    public function getIdEnfant()
    {
        return $this->idEnfant;
    }

    /**
     * @param \Enfant $idEnfant
     */
    public function setIdEnfant($idEnfant)
    {
        $this->idEnfant = $idEnfant;
    }

    /**
     * @return \TableCantine
     */
    public function getNumTable()
    {
        return $this->numTable;
    }

    /**
     * @param \TableCantine $numTable
     */
    public function setNumTable($numTable)
    {
        $this->numTable = $numTable;
    }




}

